src/shopify/product.py

```diff
@@ -64,6 +66,20 @@ def count_products(self, *args, **kwargs):
         )
         return contents["count"]
 
+    def create_product(self, **kwargs):

```

Comment:
use payload

Reply:
I'm going to change this to the payload mode, @gcandal change the places where you apply this one ;)